docs/sections/how_to_guides/basic/step/index.md

```diff
@@ -1,4 +1,4 @@
-# Step
+# Define Steps as nodes in a Pipeline
```

Comment:
Not really convinced about the title here :/

Reply:
@burtenshaw might have some other ideas.

Reply:
Personally, I think the revised title explains the content of the page clearly. And though single word titles are aesthetically appealing, and can be elegant, they are difficult to navigate if you do not already understand the package.

Furthermore, the most important attribute for this title is that there's a clear distinction between the title in the API reference on a class, and the title of a guide that uses the classes. Otherwise the docs are deliberately opaque. 

I would say that the suggested title is not perfect but it is clear improvement.

I've also noticed that HF use a similar approach on their guides vs references.

Developer Guides
![image](https://github.com/argilla-io/distilabel/assets/19620375/af43fd53-6755-41d2-a12a-8b4af139bd77)

API reference
![Screenshot from 2024-06-12 14-42-11](https://github.com/argilla-io/distilabel/assets/19620375/d3d1d002-1ed6-425e-923e-eaa459403919)

Reply:
For me the problem with the title is the word `nodes`, which we don't mention too much and that we don't use the `Pipeline` at all in this page. We just mention `Pipeline` at the start, but what we're really defining is the `Step`, not including the `Step` in a `Pipeline` (which is what I think when I read that title).

Reply:
@gabrielmbmb @davidberenstein1957 What about something simpler like 'Define the Steps of a Pipeline'?